config/config.go

```diff
@@ -111,7 +112,10 @@ func Load(path string) (config *Config, err error) {
 		return
 	}
 	config = &cfgFile.Clair
-
+	if config.API == nil || config.Database == nil || config.Notifier == nil || config.Updater == nil {
```

Comment:
This isn't going to work because the default configuration is loaded first and these fields will never be nil. Instead, we might simply add a check in the database code that tries to open the database.